src/Sylius/Component/Core/Model/TaxonImageInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Model;
+
+use Sylius\Component\Resource\Model\CodeAwareInterface;
+
+interface TaxonImageInterface extends ImageInterface, CodeAwareInterface
```

Comment:
Yup, but you're introducing it to *Sylius* so you probably know what is going on here (ok, this is pretty simple 😄 ) so I think it would be better to have an author as "guy who know how to rock'n'roll with this class" than not at all ;)